src/qt/pivx/settings/forms/settingswalletoptionswidget.ui

```diff
@@ -122,7 +122,10 @@
            </spacer>
           </item>
           <item>
-           <widget class="QSpinBox" name="spinBoxStakeSplitThreshold">
+           <widget class="QDoubleSpinBox" name="spinBoxStakeSplitThreshold">
+            <property name="decimals">
+             <number>3</number>
```

Comment:
Should be using the display decimal digits stored field instead of a fixed one (Plus, the default value is 2 decimals).

Inside the `SettingsWalletOptionsWidget` class,  `ui->spinBoxStakeSplitThreshold->setDecimals(2)`